lib/src/main/proto/bio/terra/tanagra/underlay/hierarchy.proto

```diff
@@ -17,6 +17,8 @@ message Hierarchy {
   DescendantsTable descendants_table = 2;
 
   ChildrenTable children_table = 3;
+
+  PathsTable paths_table = 4;
```

Comment:
Should probably comment here that inclusion of this will result in additional generated entity attributes, and the naming convention

Reply:
Added.